airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -118,13 +133,18 @@ public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws Retr
                 maybeJobId.get(),
                 workflowState.isResetConnection());
 
-            jobCreationAndStatusUpdateActivity.reportJobStart(new ReportJobStartInput(
-                maybeJobId.get()));
+            // todo (cgardens) - sets the Job / Attempt to running? Should this be part of creating the job and
+            // attempt too? What state is the attempt in before this activitiy runs?
+            // todo (cgardens) - why is this interleaved with the sync input workflow? should declaring sync
+            // input happen after this?
+            jobCreationAndStatusUpdateActivity.reportJobStart(new ReportJobStartInput(maybeJobId.get()));
 
             final SyncOutput syncWorkflowInputs = getSyncInputActivity.getSyncWorkflowInput(getSyncInputActivitySyncInput);
 
+            // todo (cgardens) - what does this do?
```

Comment:
Nothing apart from switching a in memory variable to true. It will also add an event to our event list for being able to test it.